### DISCUSSION TITLE
video download aborts without error after upgrading from 8.1.4.0 to 8.2.0.23

### DISCUSSION THREAD

**Discussion by Ingo62a** [aclap-dev/video-downloadhelper#149] [Help and Q&A]
### Your question

I'm trying to downloads some videos from a user/pw protected site that worked nicely in the past. However, since upgrading from 8.1.4.0 to 8.2.0.23 (and also the latest beta) the download randomly stops, sits there for a second or two and then aborts without any error.

Browser Download works reliably, with the helper app it shows the above behavior. However, only when upgrading from 8.1.4.0 to the latest stable and keeps breaking with the current beta. After downgrading to 8.1.4.0 it works again ...

Any thoughts?

### Addon & CoApp version and details

Firefox 122.0.1, MacOS 14.3.1, Apple M1 MAX

**Comment by paulrouget** [comment]
Can you confirm which version of the beta you've been using? I updated today.

Is the resulting file playable?
What's its format?

In the details of the video (in the submenu after you highlight the hit), there is a core_media section. Can you unfold it and tell me what the "builder" is ?

**Comment by Ingo62a** [comment]
Hi Paul,
the beta is 8.2.0.23a1, and yes the video (mp4) is playable, just finishing early ... the builder is "HTTPMedia"

Best regards
Ingo

attaching two pictures:

core_media from details
<img width="752" alt="Bildschirmfoto 2024-02-17 um 17 53 15" src="https://github.com/aclap-dev/video-downloadhelper/assets/160332377/4273277f-ef64-4b71-83a8-65e90ef5c830">


VLC codecdetails
<img width="496" alt="Bildschirmfoto 2024-02-17 um 17 48 21" src="https://github.com/aclap-dev/video-downloadhelper/assets/160332377/0f55d3b6-6d55-4425-86a8-1732a8ce2104">

**Comment by paulrouget** [comment]
What's your coapp version?

https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Comment by Ingo62a** [comment]
I've had 2.0.10 and upgraded to 2.0.17 (beta) earlier today to validate it was not related to the coapp version being outdated ... 8.1.4.0 works with both

best regards
Ingo
